src/Plutip/Server.purs

```diff
@@ -162,8 +189,69 @@ startOgmios cfg params = liftEffect $ spawn "ogmios" ogmiosArgs
     , params.nodeConfigPath
     ]
 
-stopOgmios :: ChildProcess -> Aff Unit
-stopOgmios = liftEffect <<< kill SIGINT
+stopChildProcess :: ChildProcess -> Aff Unit
+stopChildProcess = liftEffect <<< kill SIGINT
+
+startPlutipServer :: PlutipConfig -> Aff ChildProcess
+startPlutipServer cfg = do
+  p <- liftEffect $ spawn "plutip-server" [ "-p", UInt.toString cfg.port ]
+    defaultSpawnOptions { detached = true }
+  void
+    $ recovering defaultRetryPolicy
+        ([ \_ _ -> pure true ])
+    $ const
+    $ stopPlutipCluster cfg
```

Comment:
I don't understand why this call is needed here. `plutip-server` can only see it's own clusters. If it hasn't been started, it will now stop.

Reply:
I don't use it to stop the cluster. I use it to make sure that the server is listening.